cmds/script.c

```diff
@@ -36,25 +36,15 @@ static int cmd_script(int argc, char *argv[])
 	char buff[SIZE_CMD_ARG_LINE];
 
 	if (argc == 1) {
-		log_error("\n%s: Arguments have to be defined", argv[0]);
+		lib_printf(CONSOLE_BOLD "\nPreinit script:");
+		lib_printf(CONSOLE_NORMAL "\n%s", (char *)script);
 		return -EINVAL;
```

Comment:
Maybe we want to return 0 in this case

Reply:
Leavings from the previous condition. 
It should return EOK. Fixed.